src/modules/commander/commander.cpp

```diff
@@ -1731,6 +1733,9 @@ int commander_thread_main(int argc, char *argv[])
 									(hrt_abstime)disarm_when_landed * 1000000);
 			}
 
+			int32_t v;
+			param_get(_param_acro_att, &v);
+			acro_att = (v != 0);
```

Comment:
is there a more standard way to handle a boolean parameter? Maybe there's a boolean parameter type that I didn't know about.